channels-rke2.yaml

```diff
@@ -2519,6 +2519,44 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.2200
     featureVersions: *featureVersions-v1
+  - version: v1.29.14+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.10.99
```

Comment:
It would not matter but for consistency with the branch could we change it to 2.9.99?

Reply:
This is the same thing we've been using for all the releases I believe... looking up a few lines in the file shows ` maxChannelServerVersion: v2.10.99` for previous entries.

Reply:
Actually I take that back, I think this originated with a typo in https://github.com/rancher/kontainer-driver-metadata/pull/1598. This is the first update that used `v2.10.99`.

This should be fixed for both this month's releases, and the ones from last month.